managed/src/main/java/com/yugabyte/yw/controllers/CertificateController.java

```diff
@@ -128,13 +129,34 @@ public Result getRootCert(UUID customerUUID, UUID rootCA) {
     }
   }
 
-  public Result list(UUID customerUUID) {
-    List<CertificateInfo> certs = CertificateInfo.getAll(customerUUID);
-    if (certs == null) {
-      return ApiResponse.error(BAD_REQUEST, "Invalid Customer UUID: " + customerUUID);
-    }
-    return ApiResponse.success(certs);
-  }
+	public Result list(UUID customerUUID) {
+		List<CertificateInfo> certs = CertificateInfo.getAll(customerUUID);
+		HashMap<String, Boolean> removable = new HashMap<String, Boolean>();
```

Comment:
`HashSet<UUID> removableUUIDs = new ..`

Reply:
Made it as String as `JsonNode cert_uuid = cert.get("uuid");` returning JsonNode and converting it to string for comparision.